ibm/resource_ibm_pi_placement_group.go

```diff
@@ -0,0 +1,153 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	models "github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+)
+
+const (
+	PIPlacementGroupID      = "placement_group_id"
+	PIPlacementGroupMembers = "members"
+)
+
+func resourceIBMPIPlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPIPlacementGroupCreate,
+		ReadContext:   resourceIBMPIPlacementGroupRead,
+		UpdateContext: resourceIBMPIPlacementGroupUpdate,
+		DeleteContext: resourceIBMPIPlacementGroupDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			helpers.PIPlacementGroupName: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the placement group",
+			},
+
+			helpers.PIPlacementGroupPolicy: {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
Can we add validation here to check the Valid values are `affinity` and `anti-affinity`

Reply:
Done.